server/pkg/project/builder.go

```diff
@@ -84,3 +86,8 @@ func (b *Builder) Publication(publication *Publication) *Builder {
 	b.p.publication = publication
 	return b
 }
+
+func (b *Builder) RequestRoles(requestRoles []user.Role) *Builder {
+	b.p.requestRoles = requestRoles
```

Comment:
```suggestion
	b.p.requestRoles = slices.Clone(requestRoles)
```

I have made this same point sometimes. Do you understand why this is necessary?

Reply:
Yes, I understand why this is necessary. I am sorry about forgetting to add `slices.Clone()`.
Done!